### PR TITLE
Add color interpolation for Dwarven Hud

### PR DESCRIPTION
**Pull request by kevinthegreat1**
Tested.

### CODE REVIEW

**Comment by viciscat** [comment]
a screenshot maybe? :)

**Comment by kevinthegreat1** [comment]
Forgor... Small change and that code is already used anyways.

File: src/main/java/de/hysky/skyblocker/skyblock/tabhud/util/Colors.java [hunk 1]
```diff
@@ -10,18 +9,6 @@ public class Colors {
...
-        if (pcnt < 25) {
-            return Formatting.RED;
-        } else if (pcnt < 50) {
-            return Formatting.GOLD;
-        } else if (pcnt < 75) {
-            return Formatting.YELLOW;
-        } else {
-            return Formatting.GREEN;
-        }
+        return MathHelper.hsvToRgb(pcnt / 300, 1, 1);
```
Comment (AzureAaron) [inline src/main/java/de/hysky/skyblocker/skyblock/tabhud/util/Colors.java:12]: why change it to 1?

**Review by AzureAaron — APPROVED** [review]

File: src/main/java/de/hysky/skyblocker/skyblock/tabhud/util/Colors.java (hunk 1 above)
Comment (kevinthegreat1) [reply src/main/java/de/hysky/skyblocker/skyblock/tabhud/util/Colors.java:12]: To make it the same with vanilla colors at either end. (Red is 255, 0, 0 and green is 0, 255, 0.) Also durability also uses this gradient.

**Pull request merged**